lib/Federation/BackendNotifier.php

```diff
@@ -497,12 +501,55 @@ public function retrySendingFailedNotifications(\DateTimeInterface $dueDateTime)
 	}
 
 	protected function retrySendingFailedNotification(RetryNotification $retryNotification): void {
+		$data = json_decode($retryNotification->getNotification(), true, flags: JSON_THROW_ON_ERROR);
+		if ($retryNotification->getNotificationType() === FederationManager::NOTIFICATION_ROOM_MODIFIED) {
+			$localToken = $data['remoteToken'];
+
+			try {
+				$manager = \OCP\Server::get(Manager::class);
+				$room = $manager->getRoomByToken($localToken);
+			} catch (RoomNotFoundException) {
+				// Room was deleted in the meantime
+				return;
+			}
+
+			if ($data['changedProperty'] === ARoomModifiedEvent::PROPERTY_LOBBY) {
+				$dateTime = $room->getLobbyTimer();
+				$data['newValue'] = $room->getLobbyState();
+				$data['dateTime'] = (string)$dateTime?->getTimestamp();
+			} elseif ($data['changedProperty'] === ARoomModifiedEvent::PROPERTY_ACTIVE_SINCE) {
+				if ($room->getActiveSince() === null) {
+					$data['newValue'] = null;
+					$data['callFlag'] = Participant::FLAG_DISCONNECTED;
+				} else {
+					$data['newValue'] = $room->getActiveSince()->getTimestamp();
+					$data['callFlag'] = Participant::FLAG_DISCONNECTED;
```

Comment:
Is the callFlag still disconnected here?

Reply:
No, good catch